### PR TITLE
Update Particle Systems(2D) page

### PR DESCRIPTION
**Pull request by skyace65**
Updates the Particle Systems(2D) page for Godot 3.0.

### CODE REVIEW

File: learning/features/2d/particle_systems_2d.rst [hunk 1]
```diff
@@ -37,11 +49,111 @@ the relevant texture property:
...
+.. image:: img/paranim14.gif
+
+Lifetime: 4.0
+
+.. image:: img/paranim15.gif
+
+One Shot
+--------
+
+When enabled, a particles2D node will emit all of it's particles once
```
Comment (mhilbrunner) [inline learning/features/2d/particle_systems_2d.rst]: Particles2D

File: learning/features/2d/particle_systems_2d.rst [hunk 2]
```diff
@@ -124,137 +245,29 @@ high linear velocity and then stop as they fade.
...
-----------
-
-All physics parameters can be randomized. Random variables go from 0 to
-1. The formula to randomize a parameter is:
-
-::
-
-    initial_value = param_value + param_value*randomness
+The variation value sets the inital hue variation applied to each 
+particle. The Variation rand value controlls the hue variation
```
Comment (mhilbrunner) [inline learning/features/2d/particle_systems_2d.rst]: controls

File: learning/features/2d/particle_systems_2d.rst [hunk 3]
```diff
@@ -37,11 +49,111 @@ the relevant texture property:
...
+Fixed FPS
+---------
+
+This setting can be used to set the particle system to render at a fixed 
+FPS. For instance, changing the FPS to 2 will pake the particles render 
+every 2 frames a second, however it will not slow down. 
+
+Fract Delta
+-----------
+
```
Comment (mhilbrunner) [inline learning/features/2d/particle_systems_2d.rst]: Missed Fract Delta explanation?

File: learning/features/2d/particle_systems_2d.rst [hunk 4]
```diff
@@ -37,11 +49,111 @@ the relevant texture property:
...
 
-Before taking a look at the global parameters for the particle system,
-let's first see what happens when the physics variables are tweaked.
+    initial_value = param_value + param_value*randomness
+
+Fixed FPS
+---------
+
+This setting can be used to set the particle system to render at a fixed 
+FPS. For instance, changing the FPS to 2 will pake the particles render
```
Comment (mhilbrunner) [inline learning/features/2d/particle_systems_2d.rst]: make

File: learning/features/2d/particle_systems_2d.rst (hunk 1 above)
Comment (cbscribe) [reply learning/features/2d/particle_systems_2d.rst]: it's -> its

File: learning/features/2d/particle_systems_2d.rst [hunk 5]
```diff
@@ -37,11 +49,113 @@ the relevant texture property:
...
+.. image:: img/paranim14.gif
+
+Lifetime: 4.0
+
+.. image:: img/paranim15.gif
+
+One Shot
+--------
+
+When enabled, a Particles2D node will emit all of it's particles once
```
Comment (cbscribe) [inline learning/features/2d/particle_systems_2d.rst]: it's -> its

**Comment by skyace65** [comment]
I've fixed those issues, and a few missing periods I found.

File: learning/features/2d/particle_systems_2d.rst [hunk 6]
```diff
@@ -37,11 +49,113 @@ the relevant texture property:
...
+
+When enabled, a Particles2D node will emit all of it's particles once 
+and then never again.
+
+Preprocess
+----------
+
+Particle systems begin with 0 particles emitted, then start emitting.
+This can be an inconvenience when just loading a scene and systems like
+a torch, mist, etc begin emitting the moment you enter. Preprocess is
```
Comment (cbscribe) [inline learning/features/2d/particle_systems_2d.rst]: etc.

File: learning/features/2d/particle_systems_2d.rst [hunk 7]
```diff
@@ -37,11 +49,113 @@ the relevant texture property:
...
+used to let the system process a given amount of seconds before it is
```
Comment (cbscribe) [inline learning/features/2d/particle_systems_2d.rst]: amount -> number (you can't have an "amount of seconds", but you can have an "amount of time")

File: learning/features/2d/particle_systems_2d.rst [hunk 8]
```diff
@@ -37,11 +49,113 @@ the relevant texture property:
...
+actually shown the first time.
+
+Speed Scale
+-----------
+
+The speed scale has a default value of 1, and is used to adjust the 
+speed of a particle system. Lowering the value will make the particles 
+slower, increaseing the value will make the particles much faster.
```
Comment (cbscribe) [inline learning/features/2d/particle_systems_2d.rst]: increasing

File: learning/features/2d/particle_systems_2d.rst [hunk 9]
```diff
@@ -37,11 +49,113 @@ the relevant texture property:
...
+
+Explosiveness
+-------------
+
+If lifetime is 1 and there are 10 particles, it means every particle
+will be emitted every 0.1 seconds. The explosiveness parameter changes
+this, and forces particles to be emitted all together. Ranges are:
+
```
Comment (cbscribe) [inline learning/features/2d/particle_systems_2d.rst]: This wording is unclear.  *a* particle will be emitted every 0.1 seconds.

File: learning/features/2d/particle_systems_2d.rst [hunk 10]
```diff
@@ -37,11 +49,113 @@ the relevant texture property:
...
+-  0: Emit all particles together.
+-  1: Emit particles at equal interval.
+
```
Comment (cbscribe) [inline learning/features/2d/particle_systems_2d.rst]: This is backwards.

**Pull request merged**